apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -330,29 +466,70 @@ const PollWorkerScreen: React.FC<Props> = ({
             }
           />
         )}
+        {isSavingTally && (
+          <Modal content={<Loading>Saving to pollworker card…</Loading>} />
+        )}
+        {isPrintingCombinedReport && (
+          <Modal content={<Loading>Printing combined report…</Loading>} />
+        )}
+        {isConfirmingCombinedPrint && (
+          <Modal
+            content={
+              <Prose>
+                <p>
+                  Do you want to print the combined results report from the{' '}
+                  {pluralize('machine', machineMetadata.length, true)} (
+                  {machineMetadata.map((m) => m.machineId).join(', ')}) and
+                  clear data from the card?
```

Comment:
This reads slightly better to me:
```suggestion
                  erase the card?
```

Reply:
"erase the card" might imply that it's no longer a poll worker card... could sound scarier. Maybe we should be even more specific, "clear tally data from the card"?

Reply:
I like clear tally data.